.github/workflows/ci.yml

```diff
@@ -137,24 +137,24 @@ jobs:
           rustup component add rust-src
       - name: Check targets are installed correctly
         run: rustup target list --installed
-      - name: Restore cached Primes
-        id: cache-primes-restore
-        uses: actions/cache/restore@v3
-        with:
-          path: |
-            **/target
-          key: primes-${{ runner.os }}-${{ runner.arch }}-${{ github.sha }}
-          restore-keys: |
-            primes-${{ runner.os }}-${{ runner.arch }}-
-      - name: Restore cached Artifacts
-        id: cache-artifacts-restore
-        uses: actions/cache/restore@v3
-        with:
-          path: |
-            **/artifacts
-          key: artifacts-${{ runner.os }}-${{ runner.arch }}-${{ github.sha }}
-          restore-keys: |
-            artifacts-${{ runner.os }}-${{ runner.arch }}-
+      # - name: Restore cached Primes
+      #   id: cache-primes-restore
+      #   uses: actions/cache/restore@v3
+      #   with:
+      #     path: |
+      #       **/target
+      #     key: primes-${{ runner.os }}-${{ runner.arch }}-${{ github.sha }}
+      #     restore-keys: |
+      #       primes-${{ runner.os }}-${{ runner.arch }}-
+      # - name: Restore cached Artifacts
+      #   id: cache-artifacts-restore
+      #   uses: actions/cache/restore@v3
+      #   with:
+      #     path: |
+      #       **/artifacts
+      #     key: artifacts-${{ runner.os }}-${{ runner.arch }}-${{ github.sha }}
+      #     restore-keys: |
+      #       artifacts-${{ runner.os }}-${{ runner.arch }}-
```

Comment:
Isn't this commented out for testing purposes?

Reply:
This is the cash operation and ci is using too much cache and can cause the disk lack error.

Reply:
I see. Can I get a link to the CI log that is failing with that error for reference?

Reply:
https://github.com/starlay-finance/starlay-protocol-wasm/actions
I couldn't find the detailed link but there were some errors because of disk space.

Reply:
https://github.com/starlay-finance/starlay-protocol-wasm/actions/runs/6428142442/job/17454864467

Reply:
Thank you, this has nothing to do with this PR, so let's leave it at that for once.

I just don't know if that is the cause.

"No space left on device" is output with warn when saving

>Run actions/cache/save@v3
/usr/bin/tar --posix -cf cache.tzst --exclude cache.tzst -P -C /home/runner/work/starlay-protocol-wasm/starlay-protocol-wasm --files-from manifest.txt --use-compress-program zstdmt
zstd: error 70 : Write error : cannot write block : No space left on device 
/usr/bin/tar: cache.tzst: Wrote only 4096 of [1](https://github.com/starlay-finance/starlay-protocol-wasm/actions/runs/6428142442/job/17454864467#step:15:1)0240 bytes
/usr/bin/tar: Child returned status [7](https://github.com/starlay-finance/starlay-protocol-wasm/actions/runs/6428142442/job/17454864467#step:15:8)0
/usr/bin/tar: Error is not recoverable: exiting now
Warning: Failed to save: "/usr/bin/tar" failed with error: The process '/usr/bin/tar' failed with exit code 2
Warning: Cache save failed.

And I am getting an error message that there are no libraries needed.

>Run substrate-contracts-node --version
/home/runner/work/_temp/fc29[4](https://github.com/starlay-finance/starlay-protocol-wasm/actions/runs/6428142442/job/17454864467#step:17:5)[5](https://github.com/starlay-finance/starlay-protocol-wasm/actions/runs/6428142442/job/17454864467#step:17:6)23-3e3c-4284-929d-7c90b9981ff8.sh: line 1: substrate-contracts-node: command not found
Error: Process completed with exit code 127.